frontend/src/components/game/History.jsx

```diff
@@ -20,7 +20,7 @@ class History extends React.Component {
     // update scores from the db
     getScores = () => {
         axios
-            .get("/api/game/")
+            .get("http://localhost:8000/api/game/")
```

Comment:
once you merge into main you'll be able to replace this with `process.env.REACT_APP_SERVER_BASE_URL` (the definition is located in `/frontend/.env`)